### ISSUE TITLE
hasNumberOfAtoms field showing wrong numbers

### ISSUE THREAD

**Issue by NinadBhat** [pyscal/pyscal_rdf#72]
Hi,

I am trying to delete an atom in the lattice and writing it to a turtle file. The final hasNumberOfAtoms recorded in the turtle file is incorrect.

# Steps to reproduce

```python
from pyscal_rdf import KnowledgeGraph, System
kg = KnowledgeGraph()

# Create Al lattice
struct_Al = System.create.element.Al(graph=kg)

# Write initial data to ttl file
kg.write("initial.ttl", format="ttl")

# Delete one atom
struct_Al.delete(indices=[0])

# Write final data to file
kg.write("final.ttl", format="ttl")
```

# Version
pyscal3: 3.1.7
pyscal-rdf: 0.2.3

# Issue
The hasNumberOfAtoms field in initial.ttl is 4, which is correct. However, the hasNumberOfAtoms field in final.ttl is 2, which seems incorrect.

Can you kindly look into this?

Thanks

**Comment by srmnitc** [comment]
@NinadBhat Thanks for reporting, this was due to a `self.natoms` call after deleting the atoms. This will be fixed after merging in #73.

**Comment by srmnitc** [comment]
@NinadBhat This should be fixed in v 0.2.6, feel free to reopen if need. Thanks again!
